src/jsi/jsi_class.hpp

```diff
@@ -490,7 +495,11 @@ class ObjectWrap {
 
     static Internal* get_internal(JsiEnv env, const JsiObj& object)
     {
-        auto internal = object->getProperty(env, g_internal_field);
+        if (!s_js_internal_field_name) {
```

Comment:
```suggestion
        if (REALM_UNLIKELY(!s_js_internal_field_name)) {
```

I doubt is makes much difference, but might help some. And the whole point of this is perf, so might as well squeeze out as much as we can.